### PR TITLE
fix(Search): Incorrectly fetched list of accounts

### PR DESCRIPTION
**Pull request by leanhduy**
Update `users` querys in the `schema.js` (remove searchBy param) Update `Search` component
    - Add state to hold the filered users (by search string)
    - Change the placeholder UI element from `div` to `p` to fix no
    border radius on the last element
Update the client's `queries.js` for the `GET_USERS` GraphQL query Minor update the comment in `home.js`
Close P1-69

### CODE REVIEW

File: client/src/pages/search.js [hunk 1]
```diff
@@ -98,16 +115,24 @@ const Search = () => {
...
-                    users.map((u) => (
-                        <UserCard
-                            key={u.id}
-                            user={u}
-                            loggedInUser={loggedInUser?.userById}
-                        />
-                    ))}
-                <FetchMoreDetector ref={fetchMoreRef} />
-                {users.length === 0 && (
+                {searchTerm === ''
```
Comment (pybotlad) [inline client/src/pages/search.js:118]: Why don't just use only `filteredUserResult`?
Update this part.

**Review by pybotlad — CHANGES_REQUESTED** [review]
Please correct the code commented.

File: client/src/pages/search.js (hunk 1 above)
Comment (leanhduy) [reply client/src/pages/search.js:118]: Updated the code to use only the `filterUserResult` state

Comment (leanhduy) [reply client/src/pages/search.js:118]: Updated code to use only the state `filteredUserResult`

**Review by pybotlad — APPROVED** [review]
I'm happy with these changes, please merge into the `main` branch.

**Pull request merged**